master/include/comm/communicator.hpp

```diff
@@ -15,26 +15,36 @@ inline Code fifoCodes[] = {
     {2, BAMO_RESPONSE_ID},
     {3, AS_CU_EMERGENCY_SIGNAL},
     {4, MISSION_FINISHED},
-    {5, PC_ALIVE},
-    {6, STEERING_ID},
-    {7, RES_STATE},
-    {8, RES_READY}
+    {5, AS_CU_ID},
+    {6, RES_STATE},
+    {7, RES_READY}
 };
 
+inline Code fifoExtendedCodes[] = {
+    {8, STEERING_ID},
+};
+
+// FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> can2;
+
 /**
  * @brief Class that contains definitions of typical messages to send via CAN
  * It serves only as an example of the usage of the strategy pattern,
  * where the communicator is the strategy
  */
 class Communicator {
 private:
-    inline static FlexCAN_T4<CAN1, RX_SIZE_256, TX_SIZE_16> can1;
+    inline static FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> can2;
```

Comment:
maybe a comment reminding it's like this because of the car respective line, just to avoid future mistakes

Reply:
I have a better idea for this ->  more extensive documentation will generated regarding embedded software development in general, to avoid these challenges in the future